### DISCUSSION TITLE
Community chat

### DISCUSSION THREAD

**Discussion by babakfp** [denoland/deno#12669] [Q&A] [answered]
Hi
Is there a community chat available for Deno? Like a Discord group or something like that. I prefer Telegram because of how easy and accessible it is, but a Discord group can do the job too.

**Comment by TiKevin83** [comment, marked as answer]
discord.gg/deno is linked with a button at the top of the readme

**Comment by babakfp** [comment]
https://github.com/denoland/dotland/issues/1933
